demos/python_demos/image_inpainting_demo/inpainting_gui.py

```diff
@@ -0,0 +1,131 @@
+"""
+ Copyright (c) 2019-2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import cv2
+import numpy as np
+
+from inpainting import ImageInpainting
+from openvino.inference_engine import IECore
+
+class InpaintingGUI(object):
+    def __init__(self, srcImg, modelPath, device="CPU"):
+        self.wndName="Inpainting (press H for help)"
+        self.maskColor=(255,0,0)
+        self.radius=10
+        self.oldPoint=(-1,-1)
+
+        self.inpainter = ImageInpainting(IECore(),modelPath,device)
+   
+        self.img = cv2.resize(srcImg,(self.inpainter.input_width,self.inpainter.input_height))
+        self.originalImg = self.img.copy()
+        self.label = ""
+        self.mask = np.zeros((self.inpainter.input_height,self.inpainter.input_width,1),dtype=np.float32)
+
+        cv2.namedWindow(self.wndName, cv2.WINDOW_AUTOSIZE )
+        cv2.setMouseCallback(self.wndName,self.onMouse)
+        cv2.createTrackbar("Brush size",self.wndName,self.radius,30,self.onTrackBar)
+        cv2.setTrackbarMin("Brush size",self.wndName,1)
+
+        self.isHelpShown=False
+        self.isOriginalShown=False
+
+
+    def onMouse(self, event, x,y, flags, param):
+        if flags==cv2.EVENT_FLAG_LBUTTON and not self.isOriginalShown:
+            if self.oldPoint[0]!=-1:
+                cv2.line(self.mask,self.oldPoint,(x,y),1,self.radius*2)
+            cv2.circle(self.mask,(x,y),self.radius,1,cv2.FILLED)
+            self.oldPoint=(x,y)
+
+            self.updateWindow()
+        else:
+            self.oldPoint=(-1,-1)
+
+
+    def run(self):
+        self.updateWindow()
+
+        if self.img is None:
+            print("Cannot load source image\n")
+            return
+
+        key= cv2.waitKey(1)
+        while key not in (27,ord('q'),ord('Q')):
+            if key in (ord(" "),ord("\r")):
+                self.isOriginalShown = False
+                self.showInfo("Processing...")
+
+                self.img[:,:,0:1][self.mask>0]=0
+                self.img[:,:,1:2][self.mask>0]=0
+                self.img[:,:,2:3][self.mask>0]=0
```

Comment:
```suggestion
                self.img[self.mask>0] = 0
```

Reply:
won't work -  mask is [h,w,1] and img is [h,w,3]

Reply:
Right, forgot about that extra dimension in the mask. Then what about `self.image[np.squeeze(self.mask, -1) > 0] = 0`?